lieer/gmailieer.py

```diff
@@ -299,24 +305,25 @@ def push (self, args, setup = False):
 
       # get meta-data on changed messages from remote
       remote_messages = []
-      bar = tqdm (leave = True, total = len(gids), desc = 'receiving metadata')
+      self.bar_create (leave = True, total = len(gids), desc = 'receiving metadata')
 
       def _got_msgs (ms):
         for m in ms:
-          bar.update (1)
+          self.bar_update (1)
           remote_messages.append (m)
 
       self.remote.get_messages (gids, _got_msgs, 'minimal')
-      bar.close ()
+      self.bar_close ()
 
       # resolve changes
-      bar = tqdm (leave = True, total = len(gids), desc = 'resolving changes')
+      self.bar_create (leave = True, total = len(gids), desc = 'resolving changes')
       actions = []
       for rm, nm in zip(remote_messages, messages):
         actions.append (self.remote.update (rm, nm, self.local.state.last_historyId, self.force))
-        bar.update (1)
+        if not self.args.quiet and self.bar:
```

Comment:
redundant check for `not self.args.quiet`

Reply:
Fixed.